scripts/create-assets.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+function err() {
+  echo >&2 "$(tput setaf 1)ERROR: $*$(tput sgr0)"
+  exit 1
+}
+
+function info() {
+  echo "$(tput bold)$*$(tput sgr0)"
+}
+
+[[ -d tools ]] || err "'tools' directory not found, run script from bpftrace root dir"
+[[ -d man ]] || err "'man' directory not found, run script from bpftrace root dir"
+[[ -f bpftrace ]] || err "bpftrace binary not found, download the build artifact"
+
+OUT=$(mktemp -d) || err "Failed to create temp dir"
+TMP="${OUT}/tmp"
+
+echo "Using '$OUT'"
+
+
+set -e
+
```

Comment:
considering `xz`ing the binary as well, so we have bpftrace.xz as artifact.

Reply:
If we're going to compress might as well go w/ best algorithm (zstd maybe?)

Reply:
guess we could just do both. Zstd is not that common yet, will probably confuse people. If we XZ it we still have a nice saving